### ISSUE TITLE
Slideshow with transitions

### ISSUE THREAD

**Issue by acwolff** [dimsemenov/PhotoSwipe#1875]
Please add a slideshow option with transactions between the slides.

**Comment by amdad** [comment]
Crossfade all thy way! :)

**Comment by htmltiger** [comment]
Here is a basic plugin I have created until we wait for the official.

https://github.com/htmltiger/photoswipe-slideshow

**Comment by acwolff** [comment]
@htmltiger your plugin works fine as you can see in this [SlideShow plugin test album](https://andrewolff.jalbum.net/PS_JustifiedGallery_test-SlideShowPlugin/), but sadly there is no significant difference with [this test album](https://andrewolff.jalbum.net/PS_JustifiedGallery_test/) of my own implementation.

I miss the transitions in both implementations!

**Comment by andi34** [comment]
Added slideshow along with some other custom buttons here:
https://github.com/PhotoboothProject/photobooth/blob/d46286675d902dcbd3b862e951425377c114494b/src/js/photoswipe.js

If i remember right, it was the slideshow implementation from you @acwolff on PhotoSwipe v4

**Comment by acwolff** [comment]
@andi34 please give a link to an album made with your code.

**Comment by andi34** [comment]
https://photoboothproject.github.io/projects.html

**Comment by acwolff** [comment]
Thanks @andi34  what I see in your album is about the same as I see in my own implementation and in the plugin of @htmltiger, a slideshow without transitions.

I really hope that @dimsemenov will offer a flexible slideshow plugin with a number of selectable transitions!

**Comment by andi34** [comment]
Yes, like mentioned If I remember right it was the code you shared in the past

**Comment by designosis** [comment]
The lack of transition between slides is the only jarring / dealbreaking thing about this lightbox.

**Comment by dimsemenov** [comment]
Both slideshow plugin and crossfade transition is planned for the next version, not eta yet though. I'll update this after it's released hopefully.

**Comment by dimsemenov** [comment]
This is planned for v6 https://github.com/dimsemenov/PhotoSwipe/discussions/2170
